app/assets/javascripts/services/ChatStates.js

```diff
@@ -100,6 +100,10 @@ export class EngagedState {
         document.querySelectorAll('.agent-typing').forEach(e => e.remove());
     }
 
+    _removeAgentJoinsConference() {
+        document.querySelectorAll('.agent-joins-conference').forEach(e => e.remove());
```

Comment:
@tonymyers do you want to put the 'fadeOut(2000, "linear")' on this? I remember you saying it looked better. Maybe on the other methos where we remove an element too?

Reply:
After chat we decided using the fadeOut is not working correctly so we'll not use